release/ray_release/log_aggregator.py

```diff
@@ -0,0 +1,100 @@
+import re
+from typing import List
+
+TRACEBACK_PATTERN = "Traceback (most recent call last)"
+
+
+class LogAggregator:
+    def __init__(self, log: str):
+        self.log = log
```

Comment:
Just a nit, but it doesn't look like we actually need a class here (we could just have functions instead) - but fine with me